libevm/lib/evm.go

```diff
@@ -38,7 +38,7 @@ type EvmContext struct {
 	BlockNumber *hexutil.Big   `json:"blockNumber"`
 	Time        *hexutil.Big   `json:"time"`
 	BaseFee     *hexutil.Big   `json:"baseFee"`
-	Random      *common.Hash   `json:"random"`
+	Random      common.Hash    `json:"random"`
```

Comment:
This does not really change anything. Just that the default value will be the zero-hash instead of a `nil` pointer (in case "random" is omitted in the incoming JSON).  If that is what you were trying to do: good - if not, what were you trying to do? :wink:

Reply:
I expect now that `random` must be always passed from outside, so `nil` is not a case.